invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -303,7 +303,7 @@ class CommunityProfileForm extends Component {
     // vocabulary), so that on form submission, newly custom organization input
     // by the user can be identified and correctly sent to the backend.
     const organizationsNames = [];
-    initialValues.metadata.organizations.map((org) => {
+    initialValues.metadata.organizations.forEach((org) => {
```

Comment:
```suggestion
    let organizationsNames = [];
    organizationsNames = initialValues.metadata.organizations.map((org) => {
    const isNonCustomOrganization = org.id;
      if (isNonCustomOrganization) {
        this.knownOrganizations[org.name] = org.id;
      }
     return org.name;
    });
    
```

to be tested that the snippet works and doesn;t break the UI